tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml

```diff
@@ -147,12 +148,28 @@ jobs:
 
   - powershell: |
      Get-ChildItem -Path dist/*.whl | foreach {pip --disable-pip-version-check install --upgrade $_.fullname}
-     python $(Build.SourcesDirectory)\tools\ci_build\build.py --config $(BuildConfig) --build_dir $(Build.BinariesDirectory) $(UseOmp) --skip_submodule_sync --build_shared_lib --test --cmake_generator "Visual Studio 16 2019" --build_wheel --disable_rtti --use_winml  --build_shared_lib --enable_onnx_tests --enable_wcos $(additionalBuildFlags)
-   
+     python $(Build.SourcesDirectory)\tools\ci_build\build.py --config $(BuildConfig) --build_dir $(Build.BinariesDirectory) $(UseOmp) --test --enable_onnx_tests  --gen_doc validate $(additionalBuildFlags)
```

Comment:
Bunch of build flags missing. Intentional?

Reply:
Yeah. They're values that aren't used unless `--update` is specified.